wifiphisher/pywifiphisher.py

```diff
@@ -925,6 +925,9 @@ def run():
 
     time.sleep(3)
 
+    #no longer need mac_matcher
+    mac_matcher.unbind()
```

Comment:
My concern is that someone that runs `unbind` now and `get_vendor_name` later will produce an error. We can either catch that error or do as I said. 

Having one public method in a class is not bad. We may extend it more in the future.